src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -248,3 +250,107 @@ def _try_fetch(self) -> Iterable[Mount]:
             f"SELECT * FROM {escape_sql_identifier(self._schema)}.{escape_sql_identifier(self._table)}"
         ):
             yield Mount(*row)
+
+
+@dataclass
+class TableInMount:
+    is_delta: bool
+    is_parquet: bool
```

Comment:
```suggestion
    format: str
```
because we'll have `JSON`, `CSV`, ...